### PR TITLE
Add projection.fitWidth(), projection.fitHeight()

### PR DESCRIPTION
**Pull request by mfogel**
Discussion in #72.

The tests cover using these methods on two objects - one that's taller than it is wide and the other is wider than it is tall.

Cheers, let me know if there's changes I can make to help get this merged in.

### CODE REVIEW

**Comment by Fil** [comment]
lgtm; one can then read the other dimension with `path.bounds(feature)`

maybe add a test for the specific case of the composite projection `albersUsa`?

File: src/projection/fit.js [hunk 1]
```diff
@@ -26,6 +28,18 @@ export function fitExtent(projection, extent, object) {
       .translate([x, y]);
 }
 
+export function fitExtent(projection, extent, object) {
+  return fit(projection, extent, object)
+}
```
Comment (mbostock) [inline src/projection/fit.js]: What’s the benefit of having separate `fit` and `fitExtent` functions?

**Comment by mbostock** [comment]
The only issue I question I have is whether we need separate `fit` and `fitExtent` functions or if we can have a single `fitExtent` as we had before. Otherwise, looks good and thank you for the contribution!

**Comment by mfogel** [comment]
`fit` & `fitExtent` - my original thought was to avoid having 'hidden functionality' present in the public api. Meaning that calling `fitExtent` with `Infinity` as one of the arguments now unlocks different behavior. I was thinking to keep that special behavior present on `fit` but not have it exposed on `fitExtent` (nor `fitSize`)

That being said, looks like I didn't actually add anything to prevent that functionality from being passed straight through... so `fitExtent` and `fitSize` still have that hidden functionality available. Opps.

I'll go ahead and add a commit that merges `fit` into `fitExtent`, but let me know if you'd rather a solution that adds some checking for `Infinity` in `fitExtent` and `fitSize` to prevent that hidden functionality from being exposed by those functions.

**Comment by mbostock** [comment]
I separated the logic in 01dfd54caba399063b0f06c74ff3a12be551280b and merged. Thanks!

**Pull request merged**